streams/src/test/java/org/apache/kafka/streams/integration/utils/KafkaEmbedded.java

```diff
@@ -143,6 +137,13 @@ public void stop() {
             brokerList(), zookeeperConnect());
     }
 
+    @SuppressWarnings("WeakerAccess")
+    public void stopAsync() {
```

Comment:
Nit (feel free to ignore): Should `stopAsync` be _before_ `awaitStoppedAndPurge` ?

Reply:
Sure, this order was coincidental.